src/input/SimModSuite.h

```diff
@@ -357,6 +357,54 @@ struct get_second : public std::unary_function<MyMap::value_type, int>
     }
 };
 
+enum elementType {
```

Comment:
A class enum would be preferable, but then you can't do the loop later.

Reply:
Yes, should imo be enum class ElementType (big E)
You can then add
const ElementTypes[] = {ElementType::Vertex, ... };
below and use this in a ranged for loop.